blog/2023-07-27-introducing-substratus.md

```diff
@@ -0,0 +1,137 @@
+tags: [hello world, introduction, oss launch]
+---
+# Announcing Substratus: A Multi-cloud Substrate for AI
+
+We're beyond excited to announce Substratus (s8s), an open-source platform to
+manage machine learning primitives in Kubernetes.
+
+Open source machine learning models are exploding in popularity and it's nearly
+impossible for any individual or organization to keep up. We felt this exact
+pain in early 2023:
+
+* How can we easily experiment with LLMs? 👩‍🔬
+* Can I inference using my workstation's hardware and OS? 💻
+* What about fine-tuning, can I do that locally? Why do I run out of GPU
+  memory?! 📏
+* Can I stream or do I really need to download this whole dataset?! I didn't
+  plan on uninstalling Flight Simulator tonight...✈️
+* Is any part of this workflow portable or repeatable? ♻️
+
+It would be reasonable to walk away from a night of yak-shaving with a Colab
+subscription and a handful of modest wins. But your plan to deliver a secure,
+consistent, and portable ML workflow to your day job is still effectively
+🤷🏾‍♀️. Even through great effort and ample research, your ultimate strategy
+might be: **hope**, that your engineering team is satisfied with Sagemaker's
+technical choices and your CFO doesn't care about pricing. 💸
+
+There's got to be a better way.
+
+## Why Substratus?
+
+**Avoid vendor lock-in**: Machine learning workloads operate on undifferentiated
+compute primitives, abundant across all cloud platforms (CPU, GPU, storage,
+networks). GPUs may be less-available but are no-less commodity than the rest of
+the ML hardware stack. It takes *some* legwork to orchestrate these components
+to be useful in an ML context, but adopting a platform shouldn't come at the
+cost of lock-in.
+
+**Keep company data internal**: There's a tension playing out in real-time
+across organizations today. AI-assisted workers report both greater satisfaction
+in their jobs and non-trivial productivity gains. These trends will only
+accelerate. This should be a win-win for companies and employees alike, but
+there's a rub - the best models are hosted by another company - potentially a
+direct competitor. But even if they aren't a direct competitor, your company
+secrets are absolutely at risk. Operating your own internal inferencing services
+is the most secure path to realizing the business opportunity AI presents.
+
+**Rapid experimentation with the latest ML models**: Substratus makes it easy
+for developers to fine-tune, serve, and experiment with open-source ML models in
+notebooks. This can be done in minutes not hours using our off-the-shelf
+loaders, trainers, and servers. All this is possible without writing any code.
+
+**Best practices baked-into your ML artifacts**: While Substratus is
+less-opinionated than other platforms about your team's exact workflow, we
+**are** rather opinionated about keeping core parts of the development process
+on rails. Once created, Models, Datasets, and Servers are all immutable. They
```

Comment:
This is not 100% true. I should reword.

Reply:
once created the models are immutable from a substratus pov unless you directly go to GCS and start messing around